app/controllers/assessments_controller.rb

```diff
@@ -608,27 +608,25 @@ def viewFeedback
 
   def getPartialFeedback
     job_id = params["job_id"].to_i
-    resp = {}
+
     # User requested to view feedback on a score
     if job_id.nil?
       flash[:error] = "Invalid job id"
       redirect_to(action: "index") && return
     end
 
-    begin
+    resp = get_job_status(job_id)
```

Comment:
What happens now if the it throws an error here? Does it bubble up and get returned as an error to the frontend?

Reply:
yep it bubbles up— I tested by stopping tango in the middle of autograding so that get_job_status would throw an error and it gets caught